packages/web-components/src/components/ic-page-header/ic-page-header.tsx

```diff
@@ -180,12 +180,14 @@ export class PageHeader {
           ["sticky"]: sticky,
           ["sticky-desktop"]: !sticky && stickyDesktopOnly,
         }}
+        aria-label="page header"
```

Comment:
Could hard-coding this cause some problems, as will override if people are currently setting an aria-label. Maybe this should be the default if nothing already set?